### DISCUSSION TITLE
syncplay multiple files

### DISCUSSION THREAD

**Discussion by tleydxdy** [Syncplay/syncplay#463] [Archive]
right now there's quite a few "watchalong" format videos where the person record a video with timestamps so others can watch with them if they wish (kinda like reaction videos), example: https://www.youtube.com/watch?v=hK82X_pcj90

So I was thinking if syncplay can allow playing multiple files that would be neat, so people can watch the stream and watch with their friend at the same time.

A few problem that comes to mind, e.g. syncplay could manage the sync between multiple files, in which case there need to be a convenient way for user to set offset between them, or syncplay could just manage the sync of each file individually, in which case the user need to play files at the right time which might be kinda jank.

**Comment by Et0h** [comment]
If you disable shared playlists then you can sync multiple files at the same time and you can set offsets, it just isn't easy. Also, syncing with live streams relies on the player so how well it works depends on the player (and in my experience it often doesn't report the correct position from the start of the stream which causes problems). So maybe what you might want to do is experiment a bit with Syncplay and see if you can get a set-up that works and then make a guide on how to do it.

**Reply by tleydxdy** [reply]
Can you go into a bit on how to sync multiple files at the same time? I tried to open one file after another but it just replaces what is playing in the player opened by syncplay

**Reply by Et0h** [reply]
You need to disable shared playlists as this will mean that Syncplay won't change what is being played at all and it will only sync the playback position. You then need to run two instances of Syncplay which themselves a media player. As discussed, whether it works or not would depend on how the player handles the stream and if it allows for an equivalent position to what is used for the local file.
